gui.go

```diff
@@ -464,6 +464,11 @@ func (g *Gui) handleEvent(ev *termbox.Event) error {
 	}
 }
 
+// Flush updates the gui, re-drawing frames and buffers.
+func (g *Gui) Flush() error {
```

Comment:
I don't remember what I was thinking at that time 😄. I just needed for my project. Now I can only assume that I didn't want to introduce changes in other parts of code by making `flush` public function (i.e. changing everywhere from `flush` to `Flush`).

Please advise, what should be do with this.

If you ask me, I think we can keep it. For example, this function can be used to translate internal errors of `flush` to external errors with `Err` prefix. But on the other hand, we can just change the `flush` function.

Reply:
I'm not sure if it's a good idea to have a function that translates internal errors to exposed errors.  
That would mean we need to duplicated errors what makes me really think why wouldn't we just make those errors exposed in the first place?  
I think to prevent confusion we cloud better just expose this function.

Reply:
I decided to remove this function, and don't expose `Flush`. I think it should be done in a separate pull-request, because a lot of errors must be exposed as well, so end-users would be able to check for them. And I wasn't sure what was the best way to do it, as this project uses `errors` package, which I am not familiar with.